docs/product-updates/2023-06-03-lightning-fast-api-documentation-rendering.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Lightning-fast API Documentation rendering
+tags: [Improvement]
+---
+
+![improved-access-management.png](/files/changelog/bump-sh-performance-enhancements.png)
+
+Fast-loading page is a must these days, both for human users and for search engines. Thanks to the new rendering engine navigating your API documentation is now a breeze. It provides an optimal experience for all API consumers, whatever the size of a documentation (yes, any size).
+
+Performance is significantly better: the first interactive content is as fast as the blink of an eye, which also means there's no more blocking time when you first navigate the API documentation.
```

Comment:
Could we add a line about the fact that it's already been released to 100% of our documentation?

Reply:
A text proposal has been added including your feedback, feel free to give it a further example if needed